src/icon/meetingPoint.tsx

```diff
@@ -1,51 +1,44 @@
 // tslint:disable:max-line-length
 import React from 'react'
+import cc from 'classcat'
 import BaseIcon from '_utils/icon'
 import { BaseIconDefaultProps } from '_utils/icon/BaseIcon'
 import { color } from '_utils/branding'
 
 export interface MeetingPointIconProps extends Icon {
   readonly active?: boolean
+  readonly shadowed?: boolean
```

Comment:
I initially wanted to put this globally available for all icons, but most of them renders poorly with a round shadow around them.